python/torch_mlir_e2e_test/test_suite/conv.py

```diff
@@ -705,3 +705,80 @@ def forward(self, inputVec, weight):
 @register_test_case(module_factory=lambda: Conv_Transpose3dModule())
 def Conv_Transpose3dModule_basic(module, tu: TestUtils):
     module.forward(torch.randn(5, 2, 5, 6, 4), torch.randn(2, 5, 2, 2, 2))
+
+class UpSampleNearest2dStaticSize(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([1, 1, 4, 4], torch.float32, True),
```

Comment:
Is there a reason not to have all tests be fully dynamic?

Reply:
I have added 2 static and 2 dynamic tests. If you want me to add all static I can do that.

Reply:
I would just keep the 2 dynamic ones and remove the static ones. The static case should just be a special case of the dynamic case, so if it works dynamically, it should work statically.

Reply:
Done.

Reply:
Converted the static ones to dynamic ones as they were covering different cases (same scaled_h, scaled_w vs different).